### ISSUE TITLE
jsPlumb.detach is not a function

### ISSUE THREAD

**Issue by Tilogorn** [jsplumb/jsplumb#905]
[As the current docs state](http://jsplumb.github.io/jsplumb/removing.html#connections), you can detach a single connection by calling `jsPlumb.detach(conn);`. But this throws `jsPlumb.detach is not a function`. See https://jsfiddle.net/28fahtwL/

[In the answer to an old StackOverflow question](https://stackoverflow.com/a/48533334) one can find a hint to a undocumented function `jsPlumb.deleteConnection(conn);`.

Have the docs to be updated here?

**Comment by sporritt** [comment]
yes the docs are out of date in this case.

**Comment by rohya8** [comment]
Documentation is outdated, jsPlumb.detach() doesn't exist in the API anymore, instead use jsPlumb.deleteConnection()
